specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/SessionPools.json

```diff
@@ -0,0 +1,702 @@
+              "description": "The pool management type of the Session Pool.",
+              "enum": [
+                "Manual",
+                "Dynamic"
+              ],
+              "type": "string",
+              "x-ms-enum": {
+                "name": "PoolManagementType",
+                "modelAsString": true
+              }
+            },
+            "nodeCount": {
+              "description": "The number of nodes the Session Pool is using.",
+              "type": "integer",
+              "format": "int32",
+              "readOnly": true
+            },
+            "scaleConfiguration": {
+              "description": "The scale configuration of the Session Pool.",
+              "$ref": "#/definitions/ScaleConfiguration"
+            },
+            "secrets": {
+              "description": "The secrets of the Session Pool.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Secret"
+              },
+              "x-ms-identifiers": [
+                "name"
+              ]
+            },
+            "dynamicPoolConfiguration": {
+              "description": "The pool configuration if the poolManagementType is dynamic.",
+              "$ref": "#/definitions/DynamicPoolConfiguration"
+            },
+            "customContainerTemplate": {
+              "description": "The custom container configuration if the containerType is node-lts.",
+              "$ref": "#/definitions/CustomContainerTemplate"
+            },
+            "sessionNetworkConfiguration": {
+              "description": "The network configuration of the Sessions in the Session Pool.",
+              "$ref": "#/definitions/SessionNetworkConfiguration"
+            },
+            "poolManagementEndpoint": {
+              "description": "The endpoint to manage the pool.",
+              "type": "string",
+              "readOnly": true
+            },
+            "provisioningState": {
+              "description": "Provisioning state of the Session Pool.",
+              "enum": [
+                "InProgress",
+                "Succeeded",
+                "Failed",
+                "Canceled",
+                "Deleting"
+              ],
+              "type": "string",
+              "readOnly": true,
+              "x-ms-enum": {
+                "name": "SessionPoolProvisioningState",
+                "modelAsString": true
+              }
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "SessionPoolPatchProperties": {
+      "description": "Container App SessionPool patchable properties.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "SessionPool resource specific patchable properties.",
+          "type": "object",
+          "properties": {
+            "scaleConfiguration": {
+              "description": "The scale configuration of the Session Pool.",
```

Comment:
A nitpick about doc style - I don't think you should be capitalizing session pool as 'Session Pool' or 'SessionPool' everywhere - it is not a proper noun. Please aim for a clear style of writing.

Reply:
Hi @TimLovellSmith thanks, I have updated again according to your comments, LintDiff is passed, and I have updated all the "Session Pool" to simple "session pool".
Could you please take another look thanks